pages/design-tokens/typesetting/font-family.md

```diff
@@ -243,32 +243,58 @@ Your context and coding style determine how you access USWDS family tokens in co
   </div>
 </div>
 
-## Creating a custom typeface token
+## Adding fonts to USWDS
 
-Add fonts outside of the ones mentioned in [USWDS Available Fonts](#available-fonts) by creating a custom typeface token.
+If you need to use a font that isn’t included in [USWDS Available Fonts](#available-fonts), you can add a new font to your USWDS project. There are two typical scenarios for this:
 
-If you’re importing a new font from an open source font web directory, often you’ll be including JavaScript at the top of your file that loads the font and associates it with a display name. In addition to adding this JavaScript, you’ll need to make a typeface token that uses that display name and associates a fallback font stack with it. Here’s how to make the customizations to your code:
+1. [Adding a font from a hosting service](#adding-a-font-from-a-hosting-service)
+1. [Adding a self-hosted font](#adding-a-self-hosted-font)
 
-1. Tell USWDS about the font you’re using by defining a new typeface token in your [settings configuration]({{ site.baseurl }}/documentation/settings/#configuring-custom-uswds-settings). In the code example, we are using the font Lato.
+### Adding a font from a hosting service
 
-    ```sass
-    $theme-typeface-tokens: (
-      'lato': (
-        'display-name': 'Lato Web', // or other font
-        'cap-height': 364px,        // the default, leave it for now
-        'stack': 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif', // or whatever stack you want
-      ),
-    ),
-    ```
+If you’re importing a font from an open source font web directory, the steps will generally look like this:
 
-2. Then associate your new typeface token with the desired [type](#type-based-tokens){:.token} settings variable. In this example, we are associating "Lato" with the "sans" font type.
+1. In your HTML files, add a reference to the JavaScript and/or CSS files provided by the font hosting service.
 
-    ```sass
-    $theme-font-type-sans: 'lato',
-    ```
+{% include tokens/create-font-token.html %}
 
-3. It works! Now everything that uses the "sans" token will use the custom Lato font stack. The CSS will now include something like:
+### Adding a self-hosted font
+If you want to add a font that will be hosted in your project, you’ll need to:
+
+1. Copy font files into your fonts directory
+1. Configure `$theme-font-[font type]-custom-src` to:
+    1. Tell the system where to find your font files
```

Comment:
Updated in [afbe7da](https://github.com/uswds/uswds-site/pull/1972/commits/afbe7dad520201156a8c4c128b6f2b68f8666f92)

Note: I did not make this a global style because it differs from the USWDS list styles and I did not want it to affect the list component preview. Open to a global solution if a better solution exists.